packages/ui/v2/core/form/fields.tsx

```diff
@@ -208,15 +208,15 @@ const InputField = forwardRef<HTMLInputElement, InputFieldProps>(function InputF
             className={classNames(
               "h-9 border border-gray-300",
               addOnFilled && "bg-gray-100",
-              addOnLeading && "rounded-l-md border-r-0 px-3",
-              addOnSuffix && "border-l-0 px-3"
+              addOnLeading && "rounded-l-md border-r-0 bg-gray-100 px-3",
```

Comment:
Why add `bg-gray-100` specifically for addOnLeading.

I think if addOnLeading or addOnSuffix is true then addOnFilled should be set to true and then bg-gray-100 will be automatically added.

Reply:
My bad i thoguht i wasn't applied. Update the pr.